### ISSUE TITLE
errorbars not working

### ISSUE THREAD

**Issue by francilioaraujo** [mbaz/Gaston.jl#57]
I use the following code

```Julia
using Gaston
y = collect(1:20)
ye = rand(20)

plot(y,ye,"plotstyle","errorbars")
```

but I get the following error

```
ArgumentError: number of rows of each array must match (got (20,20,0))

 in typed_hcat(::Type{Any}, ::Array{Float64,1}, ::Array{Float64,1}, ::Array{Any,1}, ::Vararg{Array{Any,1},N}) at ./abstractarray.jl:1012
 in hcat(::Array{Float64,1}, ::Array{Float64,1}, ::Array{Any,1}, ::Vararg{Array{Any,1},N}) at ./array.jl:728
 in llplot() at /home/francilio/.julia/v0.5/Gaston/src/gaston_midlvl.jl:264
 in plot(::Array{Float64,1}, ::Vararg{Any,N}) at /home/francilio/.julia/v0.5/Gaston/src/gaston_hilvl.jl:234
```

I looked in the source code and solved with this

```
if isempty(i.ylow)
    writedlm(f,[i.x i.y],' ')
else
    writedlm(f,[i.x i.y i.ylow],' ')
end
```

on line 264 of file `gaston_midlvl.jl`

**Comment by mbaz** [comment]
Thanks for the report; I'll take a look. This reminds me that I need to add test cases to the errorbar plots.

**Comment by mbaz** [comment]
Fixed in v0.7.0.
